exercises/if/README.md

```diff
@@ -0,0 +1,12 @@
+# If
+Until now programs ran from the beginning to end following a single logic path.\
```

Comment:
```suggestion
Until now, programs ran from the beginning to end following a single logic path.\
```

Reply:
Fixed in 81bde052c49ae152c13272cd1e2548bb05859b69